sew.lic

```diff
@@ -242,9 +261,15 @@ class Sew
         next_command = "cut my #{@noun} with my scissors"
       end
     when 'Roundtime'
-      tool = 'sewing needles'
+      case lighten
```

Comment:
This might read a little clearer as:
```
      if lighten
        tool = 'scissors'
        next_command = "cut my #{@noun} with my scissors"
      else
        tool = 'sewing needles'
      end
```